src/odemis/driver/test/actuator_test.py

```diff
@@ -908,11 +910,70 @@ def test_reference(self):
         self.assertTrue(self.dev.referenced.value[axis_name])
 
     def tearDown(self):
-        axis_name = self.dev.axes.keys()[0]
         self.dev.terminate()
         super(TestCombinedFixedPositionActuator, self).tearDown()
 
 
+class TestCombinedFixedPositionActuatorCycle(unittest.TestCase):
+    """Test position at [0.0, 0.0] to test a complete rotation (cycle)
+    is handled correctly. Positions close to 2pi need to be identified
+    close to zero position."""
+
+    def setUp(self):
+        self.axis1 = "qwp"
+        self.axis2 = "linear"
+        self.axis_name = "pol"
+        self.atol = [3.392e-5, 3.392e-5]
+        self.cycle = [math.pi * 2, math.pi * 2]
+        self.fallback = "unspecified"
+        self.positions = {
+                         # [qwp, linear]
+                         # pos (str) -> list(pos (float), pos (float))
+                         "horizontal": [0.0, 0.0],
+                         "vertical": [1.570796, 1.570796],  # (pi/2, pi/2)
+                         "posdiag": [0.785398, 0.785398],  # (pi/4, pi/4)
+                         "negdiag": [2.356194, 2.356194],  # (3pi/4, 3pi/4)
+                         "rhc": [0.0, 0.785398],  # (0, pi/4)
+                         "lhc": [0.0, 2.356194],  # (0, 3pi/4)
+                         "pass-through": [1.6, 1.6],  # 91.67 degree: choose something close to vertical
+                                                      # as it will fit most real samples best
+                        }
+
+        # create one child
+        self.child1 = tmcm.TMCLController("rotstage1", "test", port="/dev/fake6",
+                                          axes=[self.axis1, self.axis2], ustepsize=[3.392e-5, 3.392e-5],
+                                          unit=["rad", "rad"],
+                                          refproc="Standard")
+
+        self.dev = CombinedFixedPositionActuator("combinedstage", "stage",
+                                                 children={"axis1": self.child1, "axis2": self.child1},
+                                                 axis_name=self.axis_name,
+                                                 caxes_map=[self.axis1, self.axis2],
+                                                 positions=self.positions,
+                                                 atol=self.atol,
+                                                 cycle=self.cycle,
+                                                 fallback=self.fallback)
+
+    def test_moveAbs(self):
+        """test position close to 2pi and therefore
+        also close to zero"""
+
```

Comment:
Please, start the test by moving the pol-analyser to a position _not_ `horizontal`.